### DISCUSSION TITLE
Remove the automatic starred function

### DISCUSSION THREAD

**Discussion by nilsbtr** [community/community#39549] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't really like the behavior that all repositories will get starred if you put them in a list, even if they are your own ones. I really like to share my liked repositories from other in my starred list, not my own ones. So I would love to see the removal of the feature.

**Comment by jfullstackdev** [comment]
but others are enjoying that too, so when can't simply ask to remove it

what you can do is to simply group them, like my personal repos starred list and other projects that you like

**Reply by ghost** [reply]
https://drive.google.com/file/d/1-DERBg8V1Wn3wVIywvierVgnCBIi-V1s/view?usp=drivesdk

**Reply by anasaboayham** [reply]
Tanks

**Reply by aunzaa** [reply]
> Tanks

**Reply by Vana12330** [reply]
За рік все буде видно

**Comment by Khawarizmy** [comment]
Yeh, it doesn't make sense. Why? You may ask. Well, because sometimes I wanna check the project later (And we all know that many projects are just good by the title). How can I star it if I haven't even checked it? 

My suggestions: 
1. Remove auto starring.
2. Add an Option that reads: Add to list and star.
3. Add a profile setting that reads: Auto star list items. 
4. Or simply make Lists as a separate panel next to Watch - Fork - Star - List

The fourth is the best way to go in my opinion.

**Comment by LeoBLMitz** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I don't really like the behavior that all repositories will get starred if you put them in a list, even if they are your own ones. I really like to share my liked repositories from other in my starred list, not my own ones. So I would love to see the removal of the feature.

**Reply by LeoBLMitz** [reply]
**1**

**Reply by LeoBLMitz** [reply]
> **1**

**Reply by Arda01guzel** [reply]
> > ### Select Topic Area
> > 
> > Product Feedback
> > 
> > ### Body
> > 
> > I don't really like the behavior that all repositories will get starred if you put them in a list, even if they are your own ones. I really like to share my liked repositories from other in my starred list, not my own ones. So I would love to see the removal of the feature.
> 
>
